modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -133,9 +134,29 @@ object SeqexecEngine {
       val instSim = cfg.require[Boolean]("seqexec-engine.instSim")
       val gcalSim = cfg.require[Boolean]("seqexec-engine.gcalSim")
 
-      for {
-        now <- Task(LocalDate.now)
-      } yield Settings(odbHost, now, dhsServer, dhsSim, tcsSim, instSim, gcalSim)
+
+    // TODO: Review initialization of EPICS systems
+    def initEpicsSystem(sys: EpicsSystem): Task[Unit] = Task(Option(CaService.getInstance()) match {
+          case None => throw new Exception("Unable to start EPICS service.")
+          case Some(s) => {
+            (sys.init(s)).leftMap {
+                case SeqexecFailure.SeqexecException(ex) => throw ex
```

Comment:
Yes, that is the reason. That error means that something went wrong loading the EPICS support libraries, which is a fatal error.